### PR TITLE
fix: bad hint in step 860 and other typos

### PR DESCRIPTION
**Pull request by Halu89**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

Hi, I tried to follow the instructions on [](https://contribute.freecodecamp.org/#/how-to-work-on-tutorials-that-use-coderoad) but I couldn't get the `git branch vX.X.X upstream/vX.X.X`  command to work & I'm not sure what it does. 
The change is pretty simple and should be working, but I did not find an easy way to go back to step 860 to test the change.

Feel free to decline the PR and apply the changes with the proper commits, branch names, etc.
As it is now, neither
`echo -e \n\n~~ kitty_ispsum_2.txt info ~~ >> kitty_info` nor
`echo -e \n\n~~ kitty_ispsum_2.txt info ~~ > kitty_info` make the lesson advance to the next step.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,3 +14,8 @@ Whenever a new version is created, add the new branch name and the changes here
 
 - Move setup commands from `coderoad.yaml` to `setup.sh`
 - Add creation of `.bash_history` in setup commands so CodeRoad watchers recognize it when the first command is entered
+
+## [v1.0.2]
+
+- Step 860 changes command operator from `>` to `>>`
+- Fix some typos
```
Comment (moT01) [inline CHANGELOG.md:21]:
```suggestion
```

File: tutorial.json [hunk 2]
```diff
@@ -3061,7 +3061,7 @@
           "hints": [
             "Here's an example: `cat $1 | sed 's/<pattern>/<replacement>/'`",
             "The `sed` argument should be `s/catnip/dogchow/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed 's/catnip/dogchow'\r\n```"
```
Comment (moT01) [inline tutorial.json:3064]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow'\n```"
```

File: tutorial.json [hunk 3]
```diff
@@ -3162,7 +3162,7 @@
           "content": "It didn't output anything, so it must be replacing all the instances of `catnip`. You can replace many patterns using `sed` like this: `sed 's/<pattern_1>/<replacement_1>/; s/<pattern_2>/<replacement_2>/'`. Note that you need the semi-colon between the two replacement patterns and they both need to be wrapped in the quotes. In your script, add another pattern to the `sed` command that replaces `cat` with `dog`.",
           "hints": [
             "The code looks like this: `s/cat/dog/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\r\n```"
```
Comment (moT01) [inline tutorial.json:3165]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\r\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\n```"
```

File: tutorial.json [hunk 4]
```diff
@@ -3233,12 +3233,12 @@
...
             ]
           },
-          "content": "It didn't find any so it must be replacing them all. You added two patterns as part of the `sed` in your script. Add a third that replaces all `moew` words with `woof`.",
+          "content": "It didn't find any so it must be replacing them all. You added two patterns as part of the `sed` in your script. Add a third that replaces all `meow` words with `woof`.",
           "hints": [
             "You can add another pattern to replace like before. Add a semi-colon and another pattern in the quotes of the `sed`",
             "Here's an example: `sed 's///; s///; s///'`",
             "The third pattern should be `s/meow/woof/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\n```"
+            "The `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\r\n```"
```
Comment (moT01) [inline tutorial.json:3241]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/; s/cat/dog/; s/meow/woof/'\n```"
```

File: tutorial.json [hunk 5]
```diff
@@ -3312,11 +3312,11 @@
...
           },
-          "content": "If you look closely, you can see that the `meow` part of `meowzer` on that one line didn't get replaced with `woof`. `grep` only matched the first instand of `meow` it found on that line. Add the \"global\" regex flag to all three patterns of the `sed` command in your script so it will replace all instances of any of the words.",
+          "content": "If you look closely, you can see that the `meow` part of `meowzer` on that one line didn't get replaced with `woof`. `grep` only matched the first instance of `meow` it found on that line. Add the \"global\" regex flag to all three patterns of the `sed` command in your script so it will replace all instances of any of the words.",
           "hints": [
             "Here's an example of one pattern: `s/<pattern>/<replacement>/<regex_flags>`",
             "It's the `g` flag",
-            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\n```"
+            "Your `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\r\n```"
```
Comment (moT01) [inline tutorial.json:3319]:
```suggestion
            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow/woof/g'\n```"
```

File: tutorial.json [hunk 6]
```diff
@@ -3365,7 +3365,7 @@
           "hints": [
             "The last `sed` pattern in you scrip should be `s/meow|meowzer/woof/`",
             "And you should add the `-E` flag to the `sed` command",
-            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\n```"
+            "Your `translate.sh` file should look like this:\r\n```sh\r\n#!/bin/bash\r\n\r\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\r\n```"
```
Comment (moT01) [inline tutorial.json:3368]:
```suggestion
            "Your `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed -E 's/catnip/dogchow/g; s/cat/dog/g; s/meow|meowzer/woof/g'\n```"
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Hey @Halu89, thanks for the PR

> I couldn't get the git branch vX.X.X upstream/vX.X.X command to work

All that command does is create the branch locally. The branch is needed to create the `tutorial.json` file. If you used `coderoad build`, it worked.

This looks to be some typo fixes, so it's actually not a new "version". It can use the existing `v1.0.1` branch. 

I made some suggestions to remove the info in the changelog, and all of the extra `\r`'s that were added in the `tutorial.json`. It's not your fault those got added - I don't think they would cause any problems, but I don't like them. If you add my suggestions we can merge this. You can use the 'Commit suggestion' button if you want.

**Comment by Halu89** [comment]
Hi, thanks for your answer, I committed your suggested changes.

File: tutorial.json [hunk 7]
```diff
@@ -3162,7 +3162,7 @@
           "content": "It didn't output anything, so it must be replacing all the instances of `catnip`. You can replace many patterns using `sed` like this: `sed 's/<pattern_1>/<replacement_1>/; s/<pattern_2>/<replacement_2>/'`. Note that you need the semi-colon between the two replacement patterns and they both need to be wrapped in the quotes. In your script, add another pattern to the `sed` command that replaces `cat` with `dog`.",
           "hints": [
             "The code looks like this: `s/cat/dog/`",
-            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\n```"
+            "The `translate.sh` file should look like this:\n```sh\r\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\n```"
```
Comment (moT01) [inline tutorial.json:3165]:
```suggestion
            "The `translate.sh` file should look like this:\n```sh\n#!/bin/bash\n\ncat $1 | sed 's/catnip/dogchow; s/cat/dog/'\n```"
```

**Review by moT01 — APPROVED** [review]
LGTM @Halu89 🎉 Thanks for fixing these 👍 

I missed one of `\r`'s so I added it quick before merging

**Pull request merged**